api/app/src/routes/medical/dtos/linkDTO.ts

```diff
@@ -75,7 +75,7 @@ function personToPatient(person: { id: string } & Person): PatientOnLinkDTO {
     lastName: personName && personName.family?.length ? joinName(personName.family) : "",
     dob: person.details?.birthDate ? person.details.birthDate : "",
     genderAtBirth: genderToDTO(person),
-    personalIdentifiers: [],
+    personalIdentifiers: undefined,
```

Comment:
nit: could just remove this line, it's not needed

Reply:
Fixed 🚀